### PR TITLE
Correction to code

### PR DESCRIPTION
**Pull request by markgov**
Due to the last Pr not generating the results i was expecting i have gone back to the code and done some moving around. i have moved the s3 bucket name to under the parameters as well as the targetkey and targetvalues i have not set the target key block to use the parameter values instead of the target block.

### CODE REVIEW

**Review by connormaglynn — DISMISSED** [review]

**Comment by markgov** [comment]
New template also builds the ssm Association so there is no need for the extra resource to be created

File: organisation-security/terraform/license-manager.tf [hunk 1]
```diff
@@ -71,35 +72,34 @@ resource "aws_cloudformation_stack" "oracleblts" {
...
+#     TargetRegions     = "eu-west-2"
+#     ArtifactsS3Bucket = "license-manager-artifact-bucket"
+#     MaxConcurrency    = "4"
+#     MaxErrors         = "4"
+#   }
+
+#   depends_on = [
+#     aws_cloudformation_stack.oracleblts
+#   ]
+# }
```
Comment (connormaglynn) [inline organisation-security/terraform/license-manager.tf:102]: Can we remove the commented out code 🥺 

Can always grab it back from the history if it's needed 😁

Comment (markgov) [reply organisation-security/terraform/license-manager.tf]: removed code on latest commit

File: organisation-security/terraform/cloudformation/temp2.yaml
[the remark this answers is not in the document]
Comment (markgov) [reply organisation-security/terraform/cloudformation/temp2.yaml:1]: removed file on latest commit

**Pull request merged**